apps/src/music/soundsPanel.module.scss

```diff
@@ -0,0 +1,93 @@
+@import "color.scss";
+@import "./theme/tokens-dark";
+
+.soundsPanel {
+  user-select: none;
+}
+
+%folder {
+  margin-top: 10px;
+  background-color: $gray-900;
+  overflow: hidden;
+  border-radius: 4px;
+  padding: 6px;
+}
+
+.folderFirst {
```

Comment:
I think you may be able to use the [:first-child](https://developer.mozilla.org/en-US/docs/Web/CSS/:first-child) selector here

Reply:
Good call.